cypress/integration/taskCloneTableColumn.spec.js

```diff
@@ -1,33 +1,24 @@
 'use strict';
 
 describe('Clone table app', () => {
-  before(() => {
+  beforeEach(() => {
     cy.visit('/');
+    cy.get('th:nth-child(2)').debug();
   });
 
   it('should only change number of columns in original table', () => {
     cy.get('table').children().should('have.length', 3);
-    cy.get('thead > tr').children().should('have.length', 6);
+    cy.get('thead > tr > th').should('have.length', 6);
     cy.get('thead').children().should('have.length', 1);
     cy.get('tbody').children().should('have.length', 12);
     cy.get('tfoot').children().should('have.length', 1);
   });
 
-  it('should copy second column and paste between fourth and sixth', () => {
-    cy.get('th:nth-child(2)').then(($th2) => {
-      const txt = $th2.text();
+  it('should copy second column and paste before the fifth one', () => {
+    cy.get('thead th').eq(1).should('have.text', 'Position');
+    cy.get('thead th').eq(4).should('have.text', 'Position');
 
-      cy.get('th:nth-child(5)').should(($th5) => {
-        expect($th5.text()).to.eq(txt);
-      });
-    });
-
-    cy.get('td:nth-child(2)').then(($td2) => {
-      const txt = $td2.text();
-
-      cy.get('td:nth-child(5)').should(($td5) => {
-        expect($td5.text()).to.eq(txt);
-      });
-    });
+    cy.get('tbody > tr').eq(0).find('td').eq(1).should('have.text', 'Accountant');
+    cy.get('tbody > tr').eq(0).find('td').eq(4).should('have.text', 'Accountant');
```

Comment:
This assertion also seems to be incorrect for the same reason as above. It's checking that the second and fifth cells in the first row have the same text, which should not be the case unless the fifth cell is indeed the cloned one.